test/api_test.js

```diff
@@ -0,0 +1,114 @@
+
+const assume = require('assume');
+const {client: _client, server: _server} = require('./helper');
+const fs = require('fs');
+const path = require('path');
+
+suite('API', () => {
+  let server;
+  let client; 
+  let workerConfig;
+
+  suiteSetup(async () => {
+    client = await _client;
+    server = await _server;
+  });
+
+  setup(async () => {
+    workerConfig = await new Promise((resolve, reject) => {
+      fs.readFile(path.join(__dirname, 'convert-expected.json'), (err, data) => {
```

Comment:
This will load the file on every test -- it might be a bit faster to load it once and copyDeep each time.

Reply:
Actually, it's significantly faster (twice as fast) to read the file and parse the JSON.  This makes sense because the file system caching of the OS ensures that after the first read, the subsequent reads are almost certainly coming straight from memory.  Also, the JSON.parse function has been optimized and is a native V8 function, performing on a very simple json file and creating a very simple object.  Deepcopy is a pure-js function which needs to make all sorts of checks and function calls within JS to be able to handle the quirks of JS objects.

But we're also talking about 0,173 ms vs 0,080ms in a unit test.

```json
[
  {
    "title": "loading file vs deepcopy",
    "results": [
      {
        "name": "deepcopy",
        "n": 10000,
        "total": 1622390493,
        "mean": 162239.0493,
        "p0": 153720,
        "p25": 155297.4693877551,
        "p50": 156776.5935672515,
        "p75": 160983.20138888888,
        "p95": 173602.84126984127,
        "p99": 209625.91666666666,
        "p100": 2443512
      },
      {
        "name": "loading file",
        "n": 10000,
        "total": 700477114,
        "mean": 70047.7114,
        "p0": 62816,
        "p25": 64894.65032679739,
        "p50": 66342.2235897436,
        "p75": 69949.72672064777,
        "p95": 80226.5,
        "p99": 92084,
        "p100": 2354470
      }
    ]
  }
]
```